a10_octavia/common/exceptions.py

```diff
@@ -142,6 +142,15 @@ def __init__(self, device_count):
         super(InvalidVCSDeviceCount, self).__init__(msg=msg)
 
 
+class IpAddressPartitionCollisionInProjectError(cfg.ConfigFileValueError):
+
+    def __init__(self, config_ip_part, existing_ip_part, project_id):
+        msg = ('Given IPAddress:Partition `{0}` in a10-octavia.conf is invalid. '
```

Comment:
@hthompson-a10 Let me know, if this error message is sufficient or needs more modification.

Reply:
Yes this should be sufficient